libs/database/src/lib/schemas/account.schema.ts

```diff
@@ -70,6 +70,12 @@ export class Account extends CastcleBase {
     number: string;
   };
 
+  @Prop({ type: Object })
+  geolocation?: {
+    countryCode: string;
+    continentCode: string;
+  };
```

Comment:
```suggestion
  geolocation?: Geolocation;
```
prefer creating a new class